modules/nw-sriov-configuring-operator.adoc

```diff
@@ -6,21 +6,35 @@
 [id="nw-sriov-configuring-operator_{context}"]
 = Configuring the SR-IOV Network Operator
 
-[IMPORTANT]
+.  Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components .
+
+.. Create a file named `sriovOperatorConfig.yaml` using the following YAML:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default 
+  namespace: openshift-sriov-network-operator 
+spec:
+  disableDrain: false
+  enableInjector: true
+  enableOperatorWebhook: true
+  logLevel: 2
```

Comment:
@SchSeba are these spec parameters mandatory to be set? Do any of them override default parameter values?

Reply:
In this PR https://77149--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-reference-cluster-configuration-for-vdu.html#ztp-sno-du-configuring-sriov_sno-configure-for-vdu the YAML profile applied for SNO is slightly different enableOperatorWebhook: false and logLevel: 0

Reply:
@cgoncalves @kquinn1204 This is the recommended configuration for multi node openshift.
for SNO the disableDrain MUST be true, and for ZTP they want to reduce CPU footprint so they also disable the webhooks 

we also have that if someone try to deploy the CR from the web console https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriov-network-operator.clusterserviceversion.yaml#L117

Reply:
@SchSeba thanks maybe not a biggie why the difference in LogLevel for ZTP where it is 0